### DISCUSSION TITLE
Google Summer of Code

### DISCUSSION THREAD

**Discussion by LeftTwixWand** [unoplatform/uno#5091] [Ideas]
## What would you like to be added:
Google Summer of Code is a global program focused on bringing more student developers into open source software development. I propose to register Uno Platform to this event.

Official site: [summerofcode.withgoogle.com](https://summerofcode.withgoogle.com)

## Why is this needed:
It will help to popularize Uno Platform and to involve new software engineers to contributing.

## Anything else we need to know?
Registration for organizations is closing at February 19, 2021

**Comment by LeftTwixWand** [comment]
So, what are you thinking about it?

**Reply by sasakrsmanovic** [reply]
Thanks for bringing to our attention. I will take a look at it today @LeftTwixWand

**Reply by LeftTwixWand** [reply]
Hi. Is there any news?

**Reply by LeftTwixWand** [reply]
Today is the last day for applying)

**Comment by trungnt2910** [comment]
Is this considered? I'm really hoping that the Uno Platform would participate in Google Summer of Code, this should greatly boost its popularity _and_ provide formal training/mentoring for potential contributors.

I'm looking forwards to this, next year I'm eligible for the GSoC, hoping to work with you as a participant!

**Reply by sasakrsmanovic** [reply]
Hi, Thanks for re-energizing this thread. The 2022 deadline seems to have passed as of yesterday. I subscribed to Google Summer of Code mailing list now so hopefully I will get catch it earlier to be able to coordinate this internally and provide a good experience both for students and our engineers.

**Reply by trungnt2910** [reply]
> hopefully I will get catch it earlier to be able to coordinate this internally and provide a good experience both for students and our engineers.

Thanks! Looking forwards to the 2023 GSoC!
